snapshots/health/2025-02-20/measles_deaths_historical.csv.dvc

```diff
@@ -0,0 +1,63 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: Historical Measles Deaths - United States
```

Comment:
We usually use this nomenclature for title snapshots. So, if the data product was on historical measless deaths, but you were extracting specifically data for the US, then we would use this format. But as far as I understand, the entire data product is on the US, so maybe the title could be "Historical measless deaths in the United States" (no big deal, up to you).